crates/primitives/fee/src/lib.rs

```diff
@@ -33,17 +33,16 @@ pub const FEE_TRANSFER_N_STORAGE_CHANGES: u8 = 2; // Sender and sequencer balanc
 /// Number of storage updates to actually charge for the fee transfer tx.
 pub const FEE_TRANSFER_N_STORAGE_CHANGES_TO_CHARGE: u8 = FEE_TRANSFER_N_STORAGE_CHANGES - 1; // Exclude the sequencer balance update, since it's charged once throughout the batch.
 
-// TODO: add real values here.
 // FIXME: https://github.com/keep-starknet-strange/madara/issues/330
 static VM_RESOURCE_FEE_COSTS: phf::Map<&'static str, f64> = phf_map! {
-    "n_steps" => 1_f64,
-    "pedersen_builtin" => 1_f64,
-    "range_check_builtin" => 1_f64,
-    "ecdsa_builtin" => 1_f64,
-    "bitwise_builtin" => 1_f64,
-    "poseidon_builtin" => 1_f64,
-    "output_builtin" => 1_f64,
-    "ec_op_builtin" => 1_f64,
+    "n_steps" => 0.01_f64,
+    "pedersen_builtin" => 0.32_f64,
+    "range_check_builtin" => 0.16_f64,
+    "ecdsa_builtin" => 20.48_f64,
+    "bitwise_builtin" => 0.64_f64,
+    "poseidon_builtin" => 0.32_f64,
+    "output_builtin" => 20.48_f64,
```

Comment:
I asked internaly for the value of the `output_builtin`. Leave it with whatever you want for now.

Reply:
I got confirmation that `output_builtin` has no meaning for starknet contracts.
You can safely remove it from the entries

Reply:
removed it